tasks.py

```diff
@@ -293,7 +288,7 @@ def generate_manifest(ctx, crd_options="crd:crdVersions=v1", bgp_type="native",
         layer=bgp_type
         if with_prometheus:
             layer = "prometheus-" + layer
-        res = run("kubectl kustomize config/{} > {}".format(layer, output))
+        res = run("{} kustomize config/{} > {}".format(kubectl_path, layer, output))
```

Comment:
I would use f-strings instead. Code is more readable. But this is fine as well of course.